apps/shell/app.js

```diff
@@ -46,16 +46,17 @@ var wss = new WebSocket.Server({ server: server });
 
 wss.on('connection', function connection (ws, req) {
   var match;
-  var host = process.env.DEFAULT_SSHHOST || 'localhost';
+  var host = process.env.DEFAULT_SSHHOST;
```

Comment:
I think here the default would be `DEFAULT_SSHHOST` or the host of the first valid cluster with login specified, instead of just undefined if DEFAULT_SSHHOST is not set.